pkg/discovery/discovery.go

```diff
@@ -360,10 +362,13 @@ func (d *Discoverer) discoverVolumesAtPath(class string, config common.MountConf
 				discoErrors = append(discoErrors, fmt.Errorf("path %q of filesystem mode cannot be used to create block volume", filePath))
 				continue
 			}
-			// Validate that this path is an actual mountpoint
-			if _, isMntPnt := mountPointMap[filePath]; isMntPnt == false {
-				discoErrors = append(discoErrors, fmt.Errorf("path %q is not an actual mountpoint", filePath))
-				continue
+			// TODO(mauriciopoppe): skipping the mountpoint verification in Windows, unlike Linux there's no /proc/mounts file to check
+			if runtime.GOOS != "windows" {
+				// Validate that this path is an actual mountpoint
+				if _, isMntPnt := mountPointMap[filePath]; isMntPnt == false {
+					discoErrors = append(discoErrors, fmt.Errorf("path %q is not an actual mountpoint", filePath))
+					continue
+				}
```

Comment:
seems like I did something here https://github.com/kubernetes-sigs/sig-storage-local-static-provisioner/pull/272/files#diff-10bda8021b225706da64124ae6d771ef8da346af6c535e8dc1a3cecfc7d5887fR289

but maybe it is not working well since I comment it out? Just as a reference.

Reply:
ok, this is a simple check to see if the location discovered is in the /proc/mounts file

This line

```
		mountPoints, err := d.RuntimeConfig.Mounter.List()
```

returns an empty array in k/mount-utils or k/k/utils/mount

Anyways, this check is only run in Linux because of that